### PR TITLE
Update password validation

### PR DESCRIPTION
**Pull request by DipanshKhandelwal**

### CODE REVIEW

File: utils/userValidation.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ const userRegistrationValidation = ({ name, email, password }) => {
 		error.password = 'No password specified';
 	}
 
+	if (Validator.isLength(password, { min: 5, max: 16 })) {
+		error.password = '"Length of password must be between 5 and 16"';
```
Comment (anshumanv) [inline utils/userValidation.js]: Double suites inside are not needed

**Review by anshumanv — CHANGES_REQUESTED** [review]
See comments

**Comment by anshumanv** [comment]
Also updated your comment, since #1 will be fixed when validations are added for all the routes.

File: utils/userValidation.js (hunk 1 above)
Comment (DipanshKhandelwal) [reply utils/userValidation.js]: Upadted

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Thanks @DipanshKhandelwal for recurring contributions. 💯

**Pull request merged**